crates/flux-tests/tests/neg/surface/assume00.rs

```diff
@@ -0,0 +1,20 @@
+#![feature(register_tool)]
+#![register_tool(flux)]
+
+#[flux::sig(fn(b:bool[true]))]
+pub fn assert(_b: bool) {}
+
+#[flux::sig(fn(b:bool) ensures b)]
+pub fn assume(b: bool) {
+    if !b {
+        panic!("assume fails")
+    }
+}
```

Comment:
We should add a test where the `ensures` fails

```rust
#[flux::sig(fn(x: i32) ensures x > 0)]
pub fn fail_ensures(x: i32) { }
```

or maybe

```rust
#[flux::sig(fn(b: bool) ensures b)]
pub fn bad_assume(b: bool) { }
```

Reply:
good call!